include/boost/geometry/algorithms/detail/overlay/traversal.hpp

```diff
@@ -741,158 +727,6 @@ public :
         return false;
     }
 
-    inline signed_size_type get_rank(sbs_type const& sbs,
-            linked_turn_op_info const& info) const
-    {
-        for (std::size_t i = 0; i < sbs.m_ranked_points.size(); i++)
-        {
-            typename sbs_type::rp const& rp = sbs.m_ranked_points[i];
-            if (rp.turn_index == info.turn_index
-                    && rp.operation_index == info.op_index
-                    && rp.direction == sort_by_side::dir_to)
-            {
-                return rp.rank;
-            }
-        }
-        return -1;
-    }
-
-    // Function checks simple cases, such as a cluster with two turns,
-    // arriving at the first turn, first turn points to second turn,
-    // second turn points further.
-    inline bool select_turn_from_cluster_linked(signed_size_type& turn_index,
```

Comment:
this functionality is now moved to `cluster_exit`, a struct is made, and there is one additional condition (the fix)